doc/threat_model/threat_model.md

```diff
@@ -188,6 +188,19 @@ TKey has been provisioned with:
 The configuration must have been written into the NVCM and
 locked by blowing the fuses.
 
+### unreleased
+
+#### Mitigations
+
+- USB port attacks - boot protocol:
+
+  - Firmware now exits to CPU trap state instead of exiting to an
+    eternal loop on errors.
+
+- Software attacks: 
+
+  - Access to RAM outside of physical RAM forces the CPU into a trap
+    state that requires a reboot instead of wrapping.
```

Comment:
I think we should remove "instead of wrapping" completely. Fine, that is what happened earlier and in a changelog that would also make sense. But here, I feel it is a bit out of context.

Reply:
Good suggestion. I'll change.